src/main/java/net/snowflake/ingest/utils/HttpUtil.java

```diff
@@ -220,11 +222,8 @@ public boolean retryRequest(
       }
 
       @Override
-      // The waiting time is backoff, and is
-      // the exponential of the executionCount.
       public long getRetryInterval() {
-        long interval = (1 << executionCount) * 1000; // milliseconds
-        return interval;
+        return 3000;
```

Comment:
Hi, why did we change from exponential backoff time to a constant time?

Reply:
Hi Jay, this exponential backoff was not thread safe. `executionCount` is being stored as an instance variable in `ServiceUnavailableRetryStrategy`, which is shared among all clients in possibly multiple threads. Two threads failing their request could be overwriting executionCount in the shared single instance of ServiceUnavailableRetryStrategy.

Reply:
I see, thanks Lukas! 
Can you extract 3000 in a static constant? Also maybe rename it to `getRetryIntervalMs`?

Reply:
Should we add a random jitter?

Reply:
I extracted the interval into a static constant, the method cannot be renamed because it is an override. I don't think a random jitter is required here.